src/Workspaces/Core/Portable/Storage/SQLite/v1/Interop/SqlConnection.cs

```diff
@@ -123,7 +106,7 @@ internal void Close_OnlyForUseBySqlPersistentStorage()
             _queryToStatement.Clear();
 
             // Finally close our handle to the actual DB.
-            ThrowIfNotOk(raw.sqlite3_close(_handle));
+            _handle.Dispose();
```

Comment:
could we `using var _ = _handle` at the top of this method.  i really like the idea of ensuring we always dispose regardless of what happens in the rest of the Close metohd.

Reply:
or, if these semantics are appropriate, we should doc taht it's ok if this doesn't get disposed.

Reply:
Done